src/frontend/src/handler/create_sink.rs

```diff
@@ -778,8 +778,6 @@ static CONNECTORS_COMPATIBLE_FORMATS: LazyLock<HashMap<String, HashMap<Format, V
                 ),
                 MqttSink::SINK_NAME => hashmap!(
                     Format::Plain => vec![Encode::Json],
-                    Format::Upsert => vec![Encode::Json],
```

Comment:
It seems that proto is also supported for mqtt sink. May also add `Encode::Proto` in the list.

Reply:
Done